extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/NotFoundExceptionMapper.java

```diff
@@ -45,6 +45,7 @@
 
     private volatile static List<String> servletMappings = Collections.EMPTY_LIST;
     private volatile static List<String> staticResources = Collections.EMPTY_LIST;
+    private volatile static Map<String, NonJaxRsClassMappings> nonJaxRsPaths = Collections.EMPTY_MAP;
```

Comment:
I would rather have a name that indicates that we're mapping from class names, which appears to be the case.

Reply:
Makes sense, will fix.

Reply:
Better now?